exercises/practice/microwave/.meta/config.json

```diff
@@ -1,6 +1,12 @@
 {
   "blurb": "Convert the buttons entered on the microwave panel to their timer equivalent",
-  "authors": [],
+  "authors": [
+    "gatorjuice"
+  ],
+  "contributors": [
+    "aleksandrilyin",
```

Comment:
I guess you don't need to include my contribution because that was a one-line fix. Thank you!